tests/e2e-tests/constants/testData.ts

```diff
@@ -1,10 +1,22 @@
-export const END_TO_END_TEST_FILE_NAME = 'endToEndTestData.txt.gz'
-export const END_TO_END_TEST_DATE_PREFIX = '2022/04/01'
-export const END_TO_END_TEST_DATE = '2022-04-01'
-export const END_TO_END_TEST_DATA_PATH =
-  'restricted.name[0].nameParts[0].value restricted.name[0].nameParts[1].value restricted.birthDate[0].value restricted.address[0].validFrom restricted.address[1].postalCode'
-export const END_TO_END_TEST_USER_ID =
-  'urn:uuid:03d3c6d3-6d7c-41df-bf45-c94401c96a2e'
-export const END_TO_END_TEST_SESSION_ID = '25bc52a5-3506-4d7a-8129-adc13a3152bf'
-export const END_TO_END_TEST_JOURNEY_ID = '71b8300f-8f06-40e7-a53e-a194beacd33f'
-export const END_TO_END_TEST_EVENT_ID = 'b122aa00-129b-46a9-b5f7-6b1bf07d427b'
+export const endToEndTestData: EndToEndTestDataConstants = {
```

Comment:
I think we can do away with endToEndTest as it's in the e2e suite.  Maybe this could be something like:
requestValues

Reply:
What about just "testData"? This is all the stuff that lives in the file, and also has the test file name too.

Reply:
I'm fine with that - I think they are all values we pipe in to Zendesk but you can keep it more generic if you want.

Reply:
Fixed